benchmarks/distributed/rpc/parameter_server/configurations/benchmark_configurations.json

```diff
@@ -4,5 +4,17 @@
         "ps_count": 0,
         "rpc_async_timeout": 15,
         "batch_size": 5
+    },
+    "4": {
+        "trainer_count": 2,
+        "ps_count": 1,
```

Comment:
nit: DDP launcher names such arguments as `--nnode`, `--nproce_per_node`. We probably wanna stay with the same convention, e.g., `--ntrainer`, `--nps`

https://github.com/pytorch/pytorch/blob/0362b753dba22c1ccab2f9f7ee9f6de2ffe45368/torch/distributed/launch.py#L43

Reply:
okay.